src/components/NetworkPanel/CyjsRenderer/cyjs-util.ts

```diff
@@ -47,6 +49,23 @@ export const createCyjsDataMapper = (vs: VisualStyle): CyjsDirectMapper[] => {
           },
         }
         cyStyle.push(selectedNodeMapping as CyjsDirectMapper)
+      } else if (vp.name === 'nodeLabelPosition') {
```

Comment:
add two different properties to the cyjs renderer when the vp is nodeLabelPosition